pyproject.toml

```diff
@@ -0,0 +1,92 @@
+[project]
+name = "openfisca-country_template"
+version = "7.0.0"
+description = "OpenFisca rules-as-code system for Country-Template."
+readme = "README.md"
+keywords = ["microsimulation", "tax", "benefit", "rac", "rules-as-code"]
+authors = []
+maintainers = []
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
```

Comment:
Probably not?

Reply:
Configured to allow bootstrap to change this to :: 1 - Planning during bootstrap process, see commit [b7b130f](https://github.com/openfisca/country-template/pull/139/commits/b7b130f9e6c4b019339e35ad4730e380e87df8ca)